### ISSUE TITLE
DDEV Complains about ports in use when Im not using Ports and when I change to new ones the next site finds default and New ports in use

### ISSUE THREAD

**Issue by asecondwill** [drud/ddev#4345]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Run a Diagnostic and Paste Link Here

https://gist.github.com/asecondwill/20b1e7d7198d618d90a7ec8744260b50

### Expected Behavior

Start and stop ddev using standard ports

### Actual Behavior

Ddev complains that 80 and 443 are in use.  

Failed to start starhr: Unable to listen on required ports, port 8080 is already in use,

It worked up until 2 weeks ago.  Yesterday I fixed one site by moving to 8443 and 8080, but now a second site is complaining that 443, 80, 8443 and 8080 are in use.  Previously I could run multiple DDEV sites on my machine. 

I tried to fix by switching to colima from docker, but the problem persists. 

I'm not running apache, lando mamp or anything else.  I'm on OSX Ventura.

### Steps To Reproduce

Configure a site.  try to run.  Error.  Fix by changing ports.  Start a second site. Error on both standard and fallback ports. 

### Anything else?

Nothing else

**Comment by asecondwill** [comment]
Hi Thanks

OK.  I  completely removed & uninstalled docker desktop  and then did
```
brew install docker
pwered off
colima start --cpu 4 --memory 6 --disk 100 --dns=1.1.1.1
ddev start

```
same problem.  

When I look to see what is conflicting, i get nothing

```
wwwroot git:stage ❯ sudo lsof -i :443 -sTCP:LISTEN                                                                                                                                                    ⏎ ✹
wwwroot git:stage ❯               
```                 


Could Fritzbox be causing this?  or is that really a seperate issue?

**Comment by rfay** [comment]
Fritzbox is worth solving, but has nothing to do with this issue.

*Something* is listening on those ports.  You can easily verify that something is listening by telnetting to it. `telnet localhost 443` for example. Sometimes `curl -I https://localhost:443` will tell you what's doing it.

But... you said you were having trouble with other ports, not 443. Please run the lsof against the ports that are actually having trouble.

**Comment by rfay** [comment]
I'll be happy to do a zoom call with you to help you sort this out. Let me know what times might work and what timezone you're in.

**Comment by asecondwill** [comment]
Its having trouble with 80 AND 443.  and then when i tried moving a ddev site to 8443 and 8080, that worked, for one site.  But not a second one. 

Here are results from curl:

wwwroot git:stage ❯ curl -I https://localhost:443                                                                                                                                                      ⏎ curl: (35) error:02FFF036:system library:func(4095):Connection reset by peer

and telnet

**Comment by rfay** [comment]
I'll be happy to do a zoom with you. We could do right now since you're still up, or we could do in my morning/your afternoon tomorrow. It would be easier to set this up if you ping me in [Discord](https://discord.gg/hCZFfAMc5k)

**Comment by asecondwill** [comment]
I've just "fixed" it.  I had Screen time parental controls on, because I had used it to try and disable access to an api to test what happens when the api was unavailable.  Removing the restriction didnt seem to be enough so I turned off content restriction compeltely and rebooted the machine.  Now ddev will start.  The restriction was not for localhost so I didnt twig it might be interfering.  Quite embarrasing.  I thought maybe the ventura update had broken it but no, it was me all along!  sorry to have wasted your time and thank you for sticking with me and suggestions to debug.
